lms/djangoapps/courseware/tests/test_inline_analytics_integration.py

```diff
@@ -35,62 +35,29 @@ def setUp(self):
         }
         json_analytics_data = json.dumps(analytics_data)
         self.data = {"data": json_analytics_data}
+        self.zendesk_response = ('A problem has occurred retrieving the data, to report the problem click '
+                                 '<a href="https://zendesk.com/hc/en-us/requests/new">here</a>')
 
-    @override_settings(ANALYTICS_ANSWER_DIST_URL='dummy_url')
-    def test_regular_user(self):
-
-        request = self.factory.post('', self.data)
-        request.user = self.user
-
-        response = get_analytics_answer_dist(request)
-        self.assertEquals(response.content, 'A problem has occurred retrieving the data, please report the problem.')
-
+    @override_settings(ZENDESK_URL='https://zendesk.com')
```

Comment:
I'm testing with and without that setting having a value. So either way I'd have to override it for a test; may as well just keep it contained to this file.